.env.example

```diff
@@ -32,3 +32,7 @@ VUE_APP_GOOGLE_ANALYTICS_ID=
 
 # mixpanel
 VUE_APP_MIXPANEL_PROJECT_TOKEN= # get this from the Mixpanel project dashboard
+
```

Comment:
need to update `docs/ENV.md`

also, deployment would definitely fail - you need to update Dockerfile and both the staging and prod github .yml files.